manuscript/01_introduction.md

```diff
@@ -14,41 +14,41 @@ locations throughout.
 
 Before beginning, we assume that you have a working knowledge
 of the R programming language. If not, there is a wonderful Coursera class
-by Roger Peng, [that can be found here](https://www.coursera.org/course/rprog).
+by Roger Peng [that can be found here](https://www.coursera.org/course/rprog).
 
 The entirety of the book is on GitHub [here](https://github.com/bcaffo/LittleInferenceBook).
-Please submit pull requests if you find errata! In addition the course notes can be found
-also on GitHub [here](https://github.com/bcaffo/courses/tree/master/06_StatisticalInference).
+Please submit pull requests if you find errata! In addition, the course notes can also be found
+on GitHub [here](https://github.com/bcaffo/courses/tree/master/06_StatisticalInference).
 While most code is in the book, *all* of the code for every figure and analysis in the
 book is in the R markdown files files (.Rmd) for the respective lectures.
 
-Finally, we should mention `swirl` (statistics with interactive R programming).
+Finally, we should mention `swirl` ("statistics with interactive R learning").
```

Comment:
Uses the acronym origin of `swirl` to explain its name. swirldev confirms that's the correct acronym even though it's not stated in the docs, in swirldev/swirl#282.